### PR TITLE
Adds load and excess to multitool power readout.

### PR DESCRIPTION
**Pull request by kriskog**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Skoglol
tweak: Multitool powernet readout now includes load and excess.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Makes it easier to diagnose power problems. Yes, the computer app exists but it's hard to use outside the main powernet.

### CODE REVIEW

**Comment by nicbn** [comment]
I think just Supply and Load are enough, no need for Excess

**Comment by Barhandar** [comment]
Agreed, excess is just subtraction, if your engineers can't do that in their mind they have bigger problems to worry about.

**Comment by nicbn** [comment]
Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -159,7 +159,7 @@ By design, d1 is the smallest direction and d2 is the highest
 
 	else if(W.tool_behaviour == TOOL_MULTITOOL)
 		if(powernet && (powernet.avail > 0))		// is it powered?
-			to_chat(user, "<span class='danger'>[DisplayPower(powernet.avail)] in power network.</span>")
+			to_chat(user, "<span class='danger'>Total power: [DisplayPower(powernet.avail)]\nLoad: [DisplayPower(powernet.load)]\nExcess power: [DisplayPower(src.surplus())]</span>")
```
Comment (ninjanomnom) [inline code/modules/power/cable.dm]:
```suggestion
			to_chat(user, "<span class='danger'>Total power: [DisplayPower(powernet.avail)]\nLoad: [DisplayPower(powernet.load)]\nExcess power: [DisplayPower(surplus())]</span>")
```
`src.` is unnecesary in 99% of cases

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/power/cable.dm (hunk 1 above)
Comment (kriskog) [reply code/modules/power/cable.dm]: Indeed, good call.

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**